drivers/led/Kconfig.lp5562

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2018 Workaround GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig LP5562
+	bool "LP5562 LED driver"
+	depends on I2C
+	help
+	  Enable LED driver for LP5562.
+
+	  LP5562 LED driver has 4 channels (RGBW). Each channel can drive up to
+	  25.5 mA per LED.
+
+if !HAS_DTS_I2C_DEVICE && LP5562
+
+config LP5562_DEV_NAME
+	string "LP5562 device name"
+	default "LP5562"
+	help
+	  Device name for LP5562 LED driver.
+
+config LP5562_I2C_ADDRESS
+	hex "LP5562 I2C slave address"
+	range 0x30 0x67
```

Comment:
range should be from `0x30` to `0x33`

Reply:
ups. That's a copy and paste fail... Thanks